File: passenger/src/main/java/com/modsen/passenger/model/Passenger.java
package com.modsen.passenger.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "passengers")
public class Passenger {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String name;
    private String surname;
    private String email;
    private Float rating;

    @Column(name = "phone_number")
    private String phoneNumber;
}

File: kt-passenger/src/main/kotlin/com/modsen/ktpassenger/model/Passenger.kt
package com.modsen.ktpassenger.model

import jakarta.persistence.*

@Entity
@Table(name = "passengers")
data class Passenger(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int? = null,
    var name: String,
    var surname: String,
    var email: String,
    var phoneNumber: String,
)
